src/sdk/model/o365/o365-restore-session.ts

```diff
@@ -1055,6 +1104,101 @@ export class O365RestoreSession extends Entity {
+            'Accept': Http.versionMime('application/octet-stream')
+          },
+          responseType: 'arraybuffer',
+          onDownloadProgress: downloadProgressCallback
+        }).then((response) => {
+          return new Blob([response.data], { type: 'application/octet-stream' });
+        });
+  }
+
+  /**
+   * Export a O365 Team Post as HTML file
+   * @param {string} teamId
+   * @param {string} channelId
+   * @param {O365TeamItemsExportRequest} request
+   * @param {function} downloadProgressCallback optional progress callback if user wishes to receive progress events
+   * @returns {Promise<Blob>} will return a promise with the Blob to download
+   */
+  /* istanbul ignore next: autogenerated */
+  async exportTeamPost(teamId: string, channelId: string, request: O365TeamItemsExportRequest,
+                       downloadProgressCallback?: (e: any) => void): Promise<Blob> {
+    // tslint:disable-next-line:max-line-length
+    return Iland.getHttp().post(`/o365-restore-sessions/${this.uuid}/teams/${teamId}/channels/${channelId}/posts/action/export`,
+        request.json, {
+          headers: {
+            'Accept': Http.versionMime('application/octet-stream')
+          },
+          responseType: 'arraybuffer',
+          onDownloadProgress: downloadProgressCallback
+        }).then((response) => {
+          return new Blob([response.data], { type: 'application/octet-stream' });
+        });
+  }
+
+  /**
+   * Download all O365 Team Posts as .msg file
+   * @param {string} teamId
+   * @param {string} channelId
+   * @param {function} downloadProgressCallback optional progress callback if user wishes to receive progress events
+   * @returns {Promise<Blob>} will return a promise with the Blob to download
+   */
+  /* istanbul ignore next: autogenerated */
+  async saveAllTeamPosts(teamId: string, channelId: string,
+                         downloadProgressCallback?: (e: any) => void): Promise<Blob> {
+    // tslint:disable-next-line:max-line-length
+    return Iland.getHttp().post(`/o365-restore-sessions/${this.uuid}/teams/${teamId}/channels/${channelId}/posts/action/save-all`,
+        null, {
+          headers: {
+            'Accept': Http.versionMime('application/octet-stream')
+          },
+          responseType: 'arraybuffer',
+          onDownloadProgress: downloadProgressCallback
+        }).then((response) => {
+          return new Blob([response.data], { type: 'application/octet-stream' });
+        });
+  }
+
+  /**
+   * Download a O365 Team Post as .msg file
+   * @param {string} teamId
+   * @param {string} channelId
+   * @param {O365TeamItemsExportRequest} request
+   * @param {function} downloadProgressCallback optional progress callback if user wishes to receive progress events
+   * @returns {Promise<Blob>} will return a promise with the Blob to download
+   */
+  /* istanbul ignore next: autogenerated */
+  async saveTeamPost(teamId: string, channelId: string, request: O365TeamItemsExportRequest,
```

Comment:
same here, is there a reason why this method is named/documented for one post when the endpoint seems to support multiple?

Reply:
Yup I agree. And in the documentation you can say "Download one or more O365 Team posts....."